_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,107 +210,75 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
-## Configuring ingress for gRPC
-
-The ingress controller currently doesn't support `.` characters in the `path`
-field. This is an issue for gRPC services using namespaces. In order to work
-around the issue, traffic can be directed through a common dummy service, with
-route rules set up to intercept traffic and redirect to the intended services.
+## Using Istio Routing Rules with Ingress
 
-1. Create a dummy ingress service:
-
-   ```bash
-   cat <<EOF | kubectl create -f -
-   apiVersion: v1
-   kind: Service
-   metadata:
-     name: ingress-dummy-service
-   spec:
-     ports:
-     - name: grpc
-       port: 1337
-   EOF
-   ```
-
-1. Create a catch-all ingress pointing to the dummy service:
-
-   ```bash
-   cat <<EOF | kubectl create -f -
-   apiVersion: extensions/v1beta1
-   kind: Ingress
-   metadata:
-     name: all-istio-ingress
-     annotations:
-       kubernetes.io/ingress.class: istio
-   spec:
-     rules:
-     - http:
-         paths:
-         - backend:
-             serviceName: ingress-dummy-service
-             servicePort: grpc
-   EOF
-   ```
-
-1. Create a RouteRule for each service, redirecting from the dummy service to
-   the correct gRPC service:
+Istio's routing rules can be used to achieve a greater degree of control
+when routing requests to backend services. For example, the following
+route rule splits the traffic between two versions of the httpbin service:
 
    ```bash
    cat <<EOF | istioctl create -f -
    apiVersion: config.istio.io/v1alpha2
    kind: RouteRule
    metadata:
-     name: foo-service-route
-   spec:
-     destination:
-       name: ingress-dummy-service
-     match:
-       request:
-         headers:
-           uri:
-             prefix: "/foo.FooService"
-     precedence: 1
-     route:
-     - weight: 100
-       destination:
-         name: foo-service
-   ---
-   apiVersion: config.istio.io/v1alpha2
-   kind: RouteRule
-   metadata:
-     name: bar-service-route
+     name: status-route
    spec:
      destination:
-       name: ingress-dummy-service
+       name: httpbin
      match:
+       # Optionally limit this rule to istio ingress pods only
+       source:
+         name: istio-ingress
+         labels:
+           istio: ingress
        request:
          headers:
            uri:
-             prefix: "/bar.BarService"
-     precedence: 1
+             prefix: /status/ #must match the path specified in ingress spec
+                 # if using prefix paths (/status/.*), omit the .*.
+                 # if using exact match, use exact: /status
      route:
-     - weight: 100
-       destination:
-         name: bar-service
-   EOF      
+     - labels:
+         version: v1
```

Comment:
This won't work because the `httpbin` service has no version labels. Previously I used a timeout rule as the example.

Reply:
That was just an example though.. Do you want it to be accurate?